OneSignalSDK/onesignal/src/main/java/com/onesignal/OSDeviceState.java

```diff
@@ -113,4 +115,22 @@ public String getEmailUserId() {
     public String getEmailAddress() {
         return emailAddress;
     }
+
+    public JSONObject toJSONObject() {
+        JSONObject mainObj = new JSONObject();
+
+        try {
+            mainObj.put("notification_enabled", notificationEnabled);
```

Comment:
Should be `are_notifications_enabled`

Reply:
Looks like we forgot to update the field name for `notificationEnabled` to be `areNotificationsEnabled` in a previous PR but can we do this here?

Reply:
since boolean getters add "is" or "are" prefix to the method as style convention, I didn't add the "are" to the variable. Do we want it to be called areNotificationsEnabled?

Reply:
also if we are changing to are_notifications_enabled should we change subscribed to is_subscribed? push_disabled to is_push_disabled?

Reply:
yes, let's change the boolean field to `areNotificationsEnabled` so it matches the getter method name exactly.